parsa-rajabi-ubc/Home-Together-Canada
# HTC-820: Remove PayPal temporarily

# [HTC-820](https://github.com/rachellegelden/Home-Together-Canada/issues/820)

## Summary
Removed PayPal from user workflow when creating a classified listing

## Relevant Motivation & Context
We will put this back in when we have Twila's banking info

## Testing Instructions
Make sure you can create a classified listing and search for it (don't forget it needs to be approved by the admin)

## Developer checklist prior to opening this pull request:


## Reviewer
